src/videojs-http-streaming.js

```diff
@@ -667,6 +670,19 @@ class HlsHandler extends Component {
       callback
     });
   }
+
+  // the player must be playing before calling this
+  seekToStreamTime(streamTime, callback, pauseAfterSeek = true, retryCount = 2) {
```

Comment:
I think it would be worth exposing the `playlist` property as part of the API, so that a user doesn't have to disable renditions and ensure we are on a specific one before calling the function.

Reply:
What would be the advantage of providing a playlist over using the currently loaded one? I think the API would be too complicated.

Reply:
My thought was that the times can be different depending on which playlist you are in, and if you are seeking, it's possible that you will change playlists.

As it stands, in order for a user to get a time from a specific playlist, they have to disable all other renditions before calling this function. Otherwise, they may end up with the time within a playlist that differs from the one they started on.

Reply:
I suppose we could add the playlist option and default to the current playlist if not provided